examples/machine-learning/a3-megagpu-8g/a3mega-slurm-blueprint.yaml

```diff
@@ -206,8 +206,8 @@ deployment_groups:
               enable_ops_agent: $(vars.enable_ops_agent)
               enable_nvidia_dcgm: $(vars.enable_nvidia_dcgm)
               nvidia_packages:
-              - cuda-toolkit-12-8
-              - datacenter-gpu-manager
+              - cuda-toolkit-12-9
```

Comment:
Do not update CUDA as part of this PR. Did you find it to be necessary for datacenter-gpu-manager-4-cuda12?